acme/setup.py

```diff
@@ -76,6 +83,7 @@
     include_package_data=True,
     install_requires=install_requires,
     extras_require={
+        ':python_version<"2.7.9"': py279_extras,
```

Comment:
Did you mean to put this in extras_require rather than install_requires? I don't see how it's ever going to get brought in unless somebody requests it manually.